ansible/roles/couchdb/files/default.d/README

```diff
@@ -0,0 +1,11 @@
+CouchDB default configuration files
+
+Files found under the etc/default.d directory that end with .ini are
+parsed within couchdb(1) at startup.
+
+This directory is intended for distribution-specific overrides of
+CouchDB defaults. Package maintainers should be placing overrides in
```

Comment:
be placing -> place

Reply:
I just coped it from CouchDB docker image.
But if required, I will update it accordingly.